### PR TITLE
FIX-BESTPRACTICES: azure-resource-optimization-toolkit

### PR DESCRIPTION
**Pull request by mohammed-spektra**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

File: demos/azure-resource-optimization-toolkit/azuredeploy.parameters.us.json [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
+      "value": "GEN-PASSWORD"
+    },
+    "azureAdmin": {
+      "value": "GEN-UNIQUE"
+    },
+    "location": {
+      "value": "usgovtexas"
+    },
+    "azurermPackageBaseUri": {
+      "value": "https://devopsgallerystorage.blob.core.windows.net/packages/"
```
Comment (bmoore-msft) [inline demos/azure-resource-optimization-toolkit/azuredeploy.parameters.us.json:15]: if this is required, then set this as the defaultValue in the template and remove this from here

File: demos/azure-resource-optimization-toolkit/azuredeploy.parameters.json [hunk 2]
```diff
@@ -1,15 +1,18 @@
...
+      "value": "GEN-PASSWORD"
+    },
+    "azureAdmin": {
+      "value": "GEN-UNIQUE"
+    },
+    "location": {
+      "value": "eastus2"
+    },
+    "azurermPackageBaseUri": {
+      "value": "https://devopsgallerystorage.blob.core.windows.net/packages/"
```
Comment (bmoore-msft) [inline demos/azure-resource-optimization-toolkit/azuredeploy.parameters.json:15]: if this is required, then set this as the defaultValue in the template and remove this from here

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

File: demos/azure-resource-optimization-toolkit/azuredeploy.parameters.json (hunk 2 above)
Comment (badri-spektra) [reply demos/azure-resource-optimization-toolkit/azuredeploy.parameters.json:15]: If we set this as the default value in the template, the TTK throws best practice violation error for hard coded url. Should we do it still?

File: demos/azure-resource-optimization-toolkit/azuredeploy.parameters.us.json (hunk 1 above)
Comment (badri-spektra) [reply demos/azure-resource-optimization-toolkit/azuredeploy.parameters.us.json:15]: If we set this as the default value in the template, the TTK throws best practice violation error for hard coded url. Should we do it still?

**Pull request merged**